tests/test_eval_model.py

```diff
@@ -33,11 +33,13 @@ def test_output(self):
         for i in range(1, len(scores)):
             score = ast.literal_eval(scores[i])
             # check totals
-            self.assertTrue(score['exs'] == i,
-                            "Total is incorrect")
+            self.assertEqual(score['exs'], i, "Total is incorrect")
             # accuracy should be one
-            self.assertTrue(score['accuracy'] == 1,
-                            "accuracy != 1")
+            self.assertEqual(score['accuracy'], 1, "accuracy != 1")
+            if 'rouge-1' in score:
```

Comment:
This means we're essentially not testing rouge in the unit tests. Add the appropriate `nltk.download` incantation to the circleci setup so that it's correctly tested.